day2/day2.py

```diff
@@ -32,25 +32,26 @@ def find_checksum():
                 elif count == 3 and three_check == 0:
                     three += 1
                     three_check += 1
+
     checksum = two * three
     return checksum
 
 
-def find_similar_strings():
-    for i in range(len(mylines)):
-        for j in range(i + 1, len(mylines) - 1):
+def find_similar_strings(string_lines):
+    for i in range(len(string_lines)):
+        for j in range(i + 1, len(string_lines) - 1):
             ratio_matching = []
-            ratio_matching = fuzz.ratio(mylines[i], mylines[j])
+            ratio_matching = fuzz.ratio(string_lines[i], string_lines[j])
             test = []
             test.append(ratio_matching)
-            percentage_correct = int((len(mylines[i]) - 1) / len(mylines[i]) * 100)
+            percentage_correct = int((len(string_lines[i]) - 1) / len(string_lines[i]) * 100)
             if ratio_matching == percentage_correct:
-                return mylines[j], mylines[i]
+                return string_lines[j], string_lines[i]
 
-def remove_duplicates(duplicates):
-    answer= [i for i, j in zip(*duplicates) if i == j]
-    return(answer)
 
+def remove_duplicates(duplicates):
+    answer = [i for i, j in zip(*duplicates) if i == j]
```

Comment:
is `duplicates` a `list`? Then you can just convert the list to a `set` instead

Reply:
What's the advantage of a set over a list?

Reply:
Everything! Sets are unindexed, making it a lot faster to look things up. But in this case you just use it to remove duplicates, since sets only contain unique values.

So you can do `list(set(duplicates))` to get rid of duplicates and then get back to a list, if you really need lists